### PR TITLE
Highlighting properties of object

### PR DESCRIPTION
**Pull request by AMalininHere**
This change should make code more readable.

### CODE REVIEW

File: grammars/javascript.cson [hunk 1]
```diff
@@ -363,6 +363,10 @@
     'name': 'support.constant.dom.js'
   }
   {
+    'match': '(?<=\\.)([a-zA-Z_?.$][\\w?.$]*)\\b'
```
Comment (kevinsawicki) [inline grammars/javascript.cson]: I'm a little unclear why `?` is in these character classes, I didn't think you could have question marks in property names.

**Comment by kevinsawicki** [comment]
Would you mind adding some tests for this? There is a spec folder where can see how the tests are structured.

**Comment by AMalininHere** [comment]
I wrote tests for it.

**Comment by AMalininHere** [comment]
Hi @kevinsawicki. Could we discuss these changes?

**Pull request closed without merging**